config/locales/en/helpers.yml

```diff
@@ -20,6 +20,8 @@ en:
         date_of_birth: Date of birth
       steps_client_contact_details_form:
         correspondence_address_type: Correspondence address
+      steps_case_case_type_form:
+        case_type: Enter the case type
```

Comment:
I think Carolina changed some of the copy in the figma design and this is a different heading now 😢